util/chunk/chunk.go

```diff
@@ -165,14 +172,21 @@ func (c *Chunk) MakeRef(srcColIdx, dstColIdx int) {
 }
 
 // MakeRefTo copies columns `src.columns[srcColIdx]` to `c.columns[dstColIdx]`.
-func (c *Chunk) MakeRefTo(dstColIdx int, src *Chunk, srcColIdx int) {
+func (c *Chunk) MakeRefTo(dstColIdx int, src *Chunk, srcColIdx int) error {
+	if c.sel != nil || src.sel != nil {
+		return errors.Trace(errors.New(msgErrSelNotNil))
```

Comment:
No need to trace. `errors.New()` already contains a stack.

Reply:
My fault, it has been removed.